EveripediaNetwork/ep-ui
# Fix performance issue

# Fix performance issue

_The homepage loads duplicate images on page load and some images have large sizes, these issues cause performance problems on the homepage, the images were optimized by adding a next image tag to images that don't have one and handling dynamic rendering to components where those images are located_

## How should this be tested?

1. When the homepage loads, open the Chrome debugger console
2. Click on the network tab and observe how the image loads

closes https://github.com/EveripediaNetwork/issues/issues/2402
